submissions/AndriyBerezovskiy/hooli-style-popup/index.html

```diff
@@ -0,0 +1,180 @@
+                  </li>
+                  <li class="popup__item df">
+                    <a class="popup__link df" href="#">
+                      <img class="popup__img" src="./img/code.svg" />
+                      <span>Code</span>
+                    </a>
+                  </li>
+                </ul>
+                <input
+                  type="checkbox"
+                  id="popup__input--more"
+                  class="popup__input popup__input--more"
+                />
+                <label for="popup__input--more" class="popup__icon df more">
+                  More
+                </label>
+                <div class="popup__content--more">
+                  <ul class="popup__list more__list">
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/chat.svg" />
+                        <span>Chat</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/qrcode.svg" />
+                        <span>Qrcode</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/database.svg" />
+                        <span>Database</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/gift.svg" />
+                        <span>Gift</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/printer.svg" />
+                        <span>Printer</span>
+                      </a>
+                    </li>
+                    <li class="popup__item more__item">
+                      <a class="popup__link df" href="#">
+                        <img class="popup__img" src="./img/translate.svg" />
+                        <span>Translate</span>
+                      </a>
+                    </li>
+                  </ul>
+                  <p class="popup__text">
+                    <a
+                      href="https://github.com/kottans/frontend/blob/2022_UA/contents.md"
+                      >Even more from Kottans</a
+                    >
+                  </p>
+                </div>
+              </div>
+            </div>
+          </li>
+          <li>
+            <a class="header__link" href="#">
+              <img src="./img/bell.svg" alt="go to the notifications page" />
```

Comment:
Why did you remove `alt` attribute from icons inside the popup and leave it for these images?

Reply:
Because the article you gave me about `alt` for images says that if the image is inside a button then `alt` is not needed. And in this example, this is exactly the situation plus each button is signed, that is, otherwise it will not go anywhere. And in other cases, everything is not so clear, for example, by clicking on the user, we can display various information about the user, modification of his profile ... But I probably did not understand correctly and I still need `alt` everywhere, so I correct this situation by adding `alt` attributes.

Reply:
I like the way you move) This `alt` content will have a great look in the `aria-label` on the link, but not here. The icon is a decorative image and for a decorative image, `alt` must be empty.

Reply:
>The icon is a decorative image, and for a decorative image, the alt must be empty
 - so I did it correctly and understood at first, and in this case, the icons do not need `alt`?  Is it possible to leave `alt` empty? But I will not change first, but rather ask if I am interpreting correctly)))

Reply:
leave an empty value: `alt=""`

Reply:
Changed the `alt` attributes to an empty value. I hope its good))